app/controllers/admin/topics_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Admin::TopicsController < Admin::ClassificationsController
 
+  def show
+    if request.format.json?
```

Comment:
Do you need to return something like `415 Unsupported Media Type` when the request isn't JSON?